### ISSUE TITLE
consider W3C Cube for advanced Measurements 

### ISSUE THREAD

**Issue by VladimirAlexiev** [w3c-ccg/traceability-vocab#300]
@OR13 
If #297 is not adopted as too complex, 
consider W3C Cube (`qb`). It has
- Observations that relate to StructureDefinitions and comprise a Dataset
- Dimensions that define the Cartesian space of each observation (i.e. there can be an observation for each combination of the dimensions)
- Measures that provide the observed values at each Cartesian point
- Attributes that further qualify the observation (eg measurement method, uncertainty)
- Slices that allow to fix some dimension values, and express related Observations more economically

schema:Observation is deficient in some aspects.
Eg it can't capture this because it doesn't have have multiple Measures in an Observation:
- A weather balloon is going up, and periodically takes observation of altitude, temperature, pressure, time.

**Comment by OR13** [comment]
yeah, I totally agree re `schema:Observation` it was a first attempt at getting something that works for Ag, Oil, Steel and E-Commerce.

cc @mprorock

**Comment by nissimsan** [comment]
We still haven't solved these requirements - this is a great suggestion.
Request for volunteers to pick this up.

**Comment by nissimsan** [comment]
Pending volunteers...?

**Comment by OR13** [comment]
Could use a JSON-LD example to make this more actionable.

**Comment by nissimsan** [comment]
@VladimirAlexiev, can you help out with a JSON-LD example, pls?

**Comment by nissimsan** [comment]
Recently we've been moving away from such sorts of generic frameworks. Denormalizing. From that perspective, I don't think this is a direction we should be going...
